src/routes/about/+page.svelte

```diff
@@ -0,0 +1,90 @@
+<script lang="ts">
+  import summer from '$lib/assets/people/summer-headshot.png';
+  import steph from '$lib/assets/people/steph-headshot.png';
+  import adrian from '$lib/assets/people/adrian-headshot.png';
+</script>
+
+<svelte:head>About</svelte:head>
+
+<div class="content">
+  <section class="paper">
+    <h1>Meet the Blend Team</h1>
+    <div class="people">
+      <div class="person">
+        <img src={summer} alt="Summer Kiesel headshot" style="border-color: #007b83;" />
+        <a href="mailto:summer@blendreading.com"><h3>Summer Kiesel</h3></a>
+        <p>Co-Founder</p>
+        <p>Reading Specialist</p>
+      </div>
+      <div class="person">
+        <img src={steph} alt="Stephanie Cruz headshot" style="border-color: #3b2e86;" />
+        <a href="mailto:stephanie@blendreading.com"><h3>Stephanie Cruz</h3></a>
+        <p>Co-Founder</p>
+        <p>Developer</p>
+      </div>
+      <div class="person">
+        <img src={adrian} alt="Adrian Moya headshot" style="border-color: #012a51;" />
+        <a href="mailto:adrian@blendreading.com"><h3>Adrian Moya</h3></a>
+        <p>Co-Founder</p>
+        <p>Developer</p>
+      </div>
+    </div>
+    <div class="text">
+      <p>
+        It all started with two old friends catching up over lunch. Summer and Stephanie, who have been friends since 2004, 
+        were talking about friends, family, and work. Summer, working as a reading specialist at her own tutoring business, 
+        mentioned that she was frustrated with one of the tools she used in her phonics instruction. She explained the tool: a blending board. 
+        The physical version didn't work very well for her online sessions, and the virtual options out there didn't meet her needs. 
+        Stephanie, who had just started her own software company with her friend Adrian, felt inspired and excited to build a working solution
+        that could help her friend teach kids to read. Stephanie and Adrian had always been interested in the intersection of education and technology,
+        so it was a perfect partnership. 
```

Comment:
```suggestion
      <p>
        It all started with two old friends catching up over lunch. Summer and Stephanie, who have been friends since 2004, 
        were talking about friends, family, and work. Summer, working as a reading specialist at her own tutoring business, 
        mentioned that she was frustrated with one of the tools she used in her phonics instruction. She explained the tool: a blending board. Physical boards didn't work very well for her online sessions, and the virtual options out there were clunky and missing basic features. <br>
        As luck would have it, Stephanie had just started her own software company with Adrian, and felt inspired and excited to build a solution
        that could help Summer teach kids to read. Stephanie and Adrian had always been interested in the intersection of education and technology,
        so it was a perfect partnership. 
```

Reply:
thank you !! love these edits. committing and merging